packages/console/src/assets/docs/guides/native-expo/README.mdx

```diff
@@ -169,4 +173,28 @@ const Content = () => {
 
 </Step>
 
+<Step title="Troubleshooting">
+
+For Expo projects, if you encounter the error `Unable to resolve "@logto/client/shim" from "node_modules/@logto/rn/lib/index.js"`, you can resolve it by adding the following to your `metro.config.js` file:
+
+```js
+// metro.config.js
+const config = {
+  // ...
+  resolver: {
+    unstable_enablePackageExports: true,
+  },
+};
+
+module.exports = config;
+```
+
+This error indicates that `@logto/rn` package not being able to resolve the `@logto/client/shim` module.
```

Comment:
nit: `not being able to resolve` reads casual and it's more like oral English to me here. Better change it to `cannot resolve` or `is not able to resolve`.

Reply:
updated